var.tfvars

```diff
@@ -10,11 +10,11 @@ network_name                = "<network_name>"
 
 ### OpenShift Cluster Details
 
-bastion                     = {instance_type    = "<bastion-compute-template>",   image_id    = "<image-uuid-rhel>"}
-# bastion                     = {instance_type    = "<bastion-compute-template>",   image_id    = "<image-uuid-rhel>",  fixed_ip_v4 = "<IPv4 address>"}
-bootstrap                   = {instance_type    = "<bootstrap-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 1}
-master                      = {instance_type    = "<master-compute-template>",    image_id    = "<image-uuid-rhcos>",  "count"   = 3}
-worker                      = {instance_type    = "<worker-compute-template>",    image_id    = "<image-uuid-rhcos>",  "count"   = 2}
+bastion                     = {instance_type    = "<bastion-compute-template>",   image_id    = "<image-uuid-rhel>", availability_zone = ""}
```

Comment:
Make it optional by removing it from here and keeping in the commented line below. SImilar to `fixed_ip_v4 `.

Reply:
Done.

Reply:
Can you please remove `availability_zone = ""` from here and other nodes config? Just add it to the commented line below.

Many users just edit and use this var.tfvars file. Changing this way will use default AZ for everyone if they don't notice. So better show it in commented lines.

Reply:
Moved all to commented lines.